backends/urls.py

```diff
@@ -0,0 +1,24 @@
+"""URLs module"""
+from django.conf.urls import url
+
+from social_django import views
+from social_django.urls import extra
+
+from backends.views import complete
+
+
+app_name = 'social'
+
+urlpatterns = [
```

Comment:
This isn't too crucial, but I think it would be good to either include a comment(s) indicating which URL rules we're lifting directly from the standard `social_django.urls` module, or import those rules directly and add this custom URL rule on top of them. Could look something like this:

```python
urlpatterns = (
  [pattern for pattern in social_django.urls.urlpatterns if pattern.name != 'complete'] +
  [url(r'^complete/(?P<backend>[^/]+)/$', complete, name='complete')]
)
```

Basically I think it would be helpful to be more explicit about the line between the library code and the custom code